pkg/ictt/operate.go

```diff
@@ -458,6 +459,7 @@ func TokenHomeAddCollateral(
 	remoteAddress common.Address,
 	amount *big.Int,
 ) error {
+	ux.Logger.PrintToUser("Collateralizing remote contract on the home chain")
```

Comment:
should we add action description logs to this lib? if this is the case, should we pass the logger as arg?